contributing/contributing-code.md

```diff
@@ -123,17 +123,11 @@ For further detail on integrating third-party services (e.g., fonts, embeds, etc
 
 ## Contributor License Agreement
 
-AMP requires all contributors to either sign an individual Contributor License Agreement or be covered by a corporate Contributor License Agreement in order to protect contributors and users in issues of intellectual property.
+AMP requires all contributors to agree to the OpenJSF Contributor License Agreement in order to protect contributors and users in issues of intellectual property.
 
-We recommend you handle signing/being covered by a CLA _before_ you send a PR to avoid problems, though this is not absolutely necessary until your code is ready to be merged in.
+To agree to the OpenJSF Contributor License Agreement, visit [https://cla-assistant.io/ampproject/amphtml](https://cla-assistant.io/ampproject/amphtml), read through the agreement, and click "Login with GitHub to Agree."
```

Comment:
nit: Can we modify this text to exactly match what CLA check says?

![image](https://user-images.githubusercontent.com/26553114/97214841-0c146e00-179a-11eb-86a5-fbcf56a1936b.png)

```suggestion
To agree to the OpenJSF Contributor License Agreement, visit [https://cla-assistant.io/ampproject/amphtml](https://cla-assistant.io/ampproject/amphtml), read through the agreement, and click "Sign in with GitHub to agree."
```